src/Navbar.tsx

```diff
@@ -192,15 +192,15 @@ class Navbar extends React.Component<any, State> {
                   >
                     SIGN&nbsp;IN
                   </Button>
-                  <Dialog
-                    // @ts-ignore
-                    onClose={handleCloseLoginDialog}
-                    open={showLoginDialog}
-                  >
-                    <SynapseComponents.Login
-                      sessionCallback={() => getSession()}
-                    />
-                  </Dialog>
+                  <Modal 
+                    animation={false}
+                    backdrop={'static'}
+                    onHide={handleCloseLoginDialog} 
+                    style={{'marginTop':'5%'}}
```

Comment:
Would prefer this to be a number (px) in case the width of this component changes in the future.